### DISCUSSION TITLE
ldaps configuration

### DISCUSSION THREAD

**Discussion by rsangalli338** [nuxsmin/sysPass#1813] [Q&A] [answered]
There is a way to configure ldaps authentication into the application?
I try to import the root CA on my linux of my ldaps server but seems not working and always usign ldap connection.

Thanks.
Roberto Sangalli.

**Comment by nuxsmin** [comment, marked as answer]
Hi Roberto, please try to search for this topic on the issues. It was covered a few times and seems to be related with an OS configuration.

Regards
